MechJeb2/MechJebCore.cs

```diff
@@ -71,7 +72,13 @@ public bool DeactivateControl
             {
                 MechJebCore mj = vessel.GetMasterMechJeb();
                 if (mj != null)
+                {
+                    if (value && !mj.deactivateControl)
+                    {
+                        controlledVessel.ctrlState.mainThrottle = currentThrottle;
+                    }
```

Comment:
you probably want to move this block to after the next line, otherwise you're checking mj.deactivateControl before you set it.

Reply:
Oh no I got it, its edge triggering, nevermind...